### PR TITLE
add ttl to persistedgrant table + change expiration from string to int

### PR DESCRIPTION
**Pull request by spudmashmedia**
add ttl to persistedgrant table + change expiration from string to int

### CODE REVIEW

File: tools/DynamoDB/Python/InitializeDynamoDB.py [hunk 1]
```diff
@@ -270,6 +270,18 @@ def CreateTables(prefix):
     except:
         print('-PersistedGrant table already exists')
 
+    try:
+        ttl_response = ddb.update_time_to_live(
+                            TableName= prefix + 'PersistedGrant',
```
Comment (brunnoferreira) [inline tools/DynamoDB/Python/InitializeDynamoDB.py]: The param indentation here is different to the rest of the file. Is that on purpose?

File: tools/DynamoDB/Python/dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+FROM python:2-jessie
+WORKDIR app
+COPY *.py ./
+RUN pip install boto3 --user
```
Comment (brunnoferreira) [inline tools/DynamoDB/Python/dockerfile:4]: No newline.

File: tools/DynamoDB/Python/run.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+docker build -t id4tools .
+docker run -it -v ~/.aws:/root/.aws id4tools bash
```
Comment (brunnoferreira) [inline tools/DynamoDB/Python/run.sh:4]: No newline.

**Review by brunnoferreira — APPROVED** [review]
Comments are only linting. So proceed as you wish. 👍

File: tools/DynamoDB/Python/InitializeDynamoDB.py (hunk 1 above)
Comment (spudmashmedia) [reply tools/DynamoDB/Python/InitializeDynamoDB.py]: good call

File: tools/DynamoDB/Python/dockerfile (hunk 2 above)
Comment (spudmashmedia) [reply tools/DynamoDB/Python/dockerfile:4]: added

File: tools/DynamoDB/Python/run.sh (hunk 3 above)
Comment (spudmashmedia) [reply tools/DynamoDB/Python/run.sh:4]: added

**Review by gavanlamb — APPROVED** [review]

**Pull request merged**